javascript/chrome-driver/atoms.js

```diff
@@ -259,9 +259,17 @@ webdriver.chrome.isElementClickable = function(elem, coord) {
     }
     parentElemIter = parentElemIter.parentNode;
   }
+  var elemHTML = elem.outerHTML;
+  if (elem.hasChildNodes()) {
+    var inner = elem.innerHTML;
```

Comment:
or remove the if block and just have:

    var elemHTML = elem.outerHTML.replace(elem.innerHTML, elem.hasChildNodes() ? '...' : '');